Source/WebCore/rendering/style/RenderStyle.cpp

```diff
@@ -1343,8 +1342,7 @@ bool RenderStyle::changeRequiresRepaint(const RenderStyle& other, OptionSet<Styl
 
     if (m_inheritedFlags.visibility != other.m_inheritedFlags.visibility
         || m_inheritedFlags.printColorAdjust != other.m_inheritedFlags.printColorAdjust
-        || m_inheritedFlags.insideLink != other.m_inheritedFlags.insideLink
-        || m_inheritedFlags.insideDefaultButton != other.m_inheritedFlags.insideDefaultButton)
```

Comment:
You'll need to update `rareInheritedDataChangeRequiresRepaint` to check for `insideDefaultButton` now that it is not longer checked here.